src/state/sagas/auth/signupFinishSaga.js

```diff
@@ -42,6 +42,8 @@ function* run() {
   });
 
   yield call(navigator.navigate, 'Auth', 'Ready');
+
+  trackEvent('Auth::Success');
```

Comment:
Again, probably [`logSignup`](https://github.com/corymsmith/react-native-fabric/blob/master/Answers.js#L136) is better suited for this.